examples/evm-to-evm-fungible-transfer/src/transfer.ts

```diff
@@ -21,7 +21,7 @@ export async function erc20Transfer(): Promise<void> {
     SEPOLIA_CHAIN_ID,
     await wallet.getAddress(), // Sending to the same address on a different chain
     RESOURCE_ID,
-    50
+    "50000000000000000000" // 18 decimal places
```

Comment:
what if we use `parseUnits` or `parseEther` instead of the hardcoded number?

Reply:
The use of these helpers is encouraged, but the exact implementation is entirely up to the developer integrating the SDK into their solution. This is outside the direct responsibilities of the SDK.

Reply:
got it